python-avd/pyavd/_eos_designs/schema/schema_fragments/event_monitor.schema.yml

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  event_monitor:
+    type: bool
```

Comment:
we can have `enabled` key here same as eos_cli_config_gen and check for it in code.
lets confirm with @gmuloc

Reply:
the ask from teh issue is to "add 1:1 models from the eos_cli_config_gen schema" so lets make sure the models match

Reply:
Added!